Branch-SDK/Branch-SDK/BNCPreferenceHelper.h

```diff
@@ -31,8 +31,9 @@
 @property (assign, nonatomic) BOOL explicitlyRequestedReferrable;
 @property (assign, nonatomic) BOOL isReferrable;
 @property (assign, nonatomic) BOOL isDebug;
-@property (assign, nonatomic) BOOL isContinuingUserActivity;
+@property (assign, nonatomic) BOOL shouldWaitForInit;
```

Comment:
I disagree with this name change. It's not actually about waiting for init. With this variable name, the following code reads a bit silly:

https://github.com/BranchMetrics/ios-branch-deep-linking/blob/db641cc19c0c33dd63ede59c57ddfdac7562efc2/Branch-SDK/Branch-SDK/Branch.m#L344-L350

I'm not sure what a good name is, which is why I gave it such a horrible name to begin with